app/views/partials/lib/commands/posts/create/build.liquid

```diff
@@ -6,7 +6,7 @@
     "title": {{ object.title | json }},
     "body": {{ object.body | json }},
     "tags": {{ object.tags | downcase | split: ',' | uniq }},
-    "c__score": {{ object.c__score | json }}
+    "c__score": {{ object.c__score | default: 0 | json }}
```

Comment:
Do we want to just set it to `0` instead of setting a default? We shouldn't accept input from the user for this field.